examples/private-endpoint-regional-mode/privateEndpointRegionalMode.json

```diff
@@ -0,0 +1,32 @@
+{
+  "AWSTemplateFormatVersion" : "2010-09-09",
+  "Description" : "This template creates a Project on the MongoDB Atlas API, this will be billed to your Atlas account.",
```

Comment:
This appears to be for Project resource. Please update for Private Endpoint Regional Mode